### DISCUSSION TITLE
XML template not fully working on treadmill domyos intense run

### DISCUSSION THREAD

**Discussion by ed0c** [cagnulein/qdomyos-zwift#2698] [Q&A] [answered]
Hello,
I'm trying to make workout programs using xml template for my treadmill domyos Intense run.
An example:

    <row duration="00:30:00" speed="10.5" inclination="0" fanspeed="2"/>
 

Unfortunately, speed is staying at 0kmph with the example above, but the fan is working..
So, what is the parameter to modify the speed of the treadmill ?

**Comment by cagnulein** [comment]
You missed the forcespeed parameter as in my example

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno ven 25 ott 2024 alle 21:51 ed0c ***@***.***> ha
scritto:

**Comment by ed0c** [comment]
Like this ?

    <row duration="00:30:00" speed="10.5" inclination="0" fanspeed="2" forcespeed="10.5"/>

Or this ?

    <row duration="00:30:00" forcespeed="10.5" inclination="0" fanspeed="2"/>

**Comment by cagnulein** [comment, marked as answer]
Likes this

<row duration="00:30:00" speed="10.5" inclination="0" fanspeed="2"
forcespeed="1"/>



Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno ven 25 ott 2024 alle 21:58 ed0c ***@***.***> ha
scritto:

**Comment by ed0c** [comment]
OK!!!
Thanks for the fast reply, i'll try this .
